README.md

```diff
@@ -202,6 +202,23 @@ behavior of the omnibox:
        last.
     4. Hitting ESC will close the list of suggestions and clear the field.
 
+## Omnibox Event Bindings
+
+The following &-callback functions are available for binding on the `ngc-omnibox` component in
+response to events:
+
+- `onChosen({choice})`: An expression that's called when a suggestion is chosen. It receives, in its
+scope, access to `choice`, which is the item that was chosen.
+- `onUnchosen({choice})`: An expression that's called when a suggestion is unchosen (removed as a
+choice). It receives, in its scope, access to `choice`, which is the item that was unchosen.
+- `onNgModelChange({ngModel})`: An expression that's called when the `ngModel` is updated (choices
+are added or removed). It receives, in its scope, access to the `ngModel` object with the changes
+applied.
```

Comment:
Two reasons:

1. For general API design, it's usually best to give the minimum surface area possible. Don't give the developer 2 ways to accomplish the same thing if it can be avoided. People will read documentation and jump on the first way to do it they see. Some will see the separate way first, and others will see the single change binding first. These people will have difficulty working with each other for trivial reasons. For a real world example, I've met many engineers who are unable to tell me the difference between `[].map()` and `[].forEach()`, and those two actually have valid differences!

2. I believe that there are developer expectations around `ngModel` specifically due to Angular training and learning. I believe that having both `ng-model` and `on-ng-model-change` as bindings will cause confusion because none of their other `ng-model` stuff has this other binding available. API design is a lot like UI design -- you want to build on things the user is already trained to do.